userspace/libscap/scap_savefile.c

```diff
@@ -400,18 +397,39 @@ int32_t scap_write_proclist_entry_bufs(scap_t *handle, scap_dumper_t *d, struct
 static int32_t scap_write_proclist(scap_t *handle, scap_dumper_t *d)
 {
 	uint32_t totlen = 0;
+	uint32_t nprocs = 0;
+	uint32_t idx = 0;
 	struct scap_threadinfo *tinfo;
 	struct scap_threadinfo *ttinfo;
 
 	//
-	// First pass pass of the table to calculate the length
+	// First pass of the table to calculate the size
+	//
+	HASH_ITER(hh, handle->m_proclist, tinfo, ttinfo)

```

Comment:
You should be able to use HASH_COUNT here, right?

Reply:
Oh never mind--you already fixed this in later commits.

Reply:
done